src/components/wallet/events/componentWillLoad.ts

```diff
@@ -18,6 +18,13 @@ async function loadAccountFromBrowser(wallet: Wallet) {
   } catch {
     throw 'Unable to parse query string'
   }
+  if (query.pubnet === 'true') {
+    wallet.network_passphrase = Networks.PUBLIC
+    wallet.server = new Server('https://horizon.stellar.org')
```

Comment:
In a separate PR I think we should replace these url strings with a global constant like `TESTNET_URI` and `MAINNET_URI`.